employee-management/webapp/view/Detail.view.xml

```diff
@@ -0,0 +1,76 @@
+<mvc:View
```

Comment:
The name of the view name makes more sense to keep the long name like: EmployeeDetail (view and controller)